src/main/java/org/sonarsource/scanner/maven/bootstrap/ScannerBootstrapper.java

```diff
@@ -65,7 +71,12 @@ public void execute() throws MojoExecutionException {
       scanner.start();
       serverVersion = scanner.serverVersion();
 
-      checkSQVersion();
+      if (isSonarCloudUsed()) {
+        log.info("Communicating with SonarCloud");
+      } else {
+        checkSQVersion();
```

Comment:
I think this is a good opportunity to remove the logging from `checkSQVersion` and restrict to checking that we can talk with the version.
```suggestion
        checkSQVersion();
        if (serverVersion != null) {
          log.info("Communicating with SonarQube Server " + serverVersion);
        }
```

```java
private void checkSQVersion() {
  if (isVersionPriorTo("5.6")) {
    throw new UnsupportedOperationException(UNSUPPORTED_BELOW_SONARQUBE_56_MESSAGE);
  }
}
```